src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -30,7 +30,9 @@ public Task DetailedStats()
                 }
 
                 using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
-                    writer.WriteDetailedDatabaseStatistics(context, stats);
```

Comment:
@Danielle9897 Do we have any usage of this method now?

Reply:
No usages. I will remove.